.github/workflows/helm-release.yml

```diff
@@ -2,14 +2,107 @@ name: Release Charts
 
 on:
   push:
+    branches:
+      - TECH-2871-improve-cicd-chartss
```

Comment:
Please don't forget to remove when done testing

Reply:
missed to do the commit command somehow
```
$ git add .
$ git status
On branch TECH-2871-improve-cicd-charts
Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	modified:   .github/workflows/helm-release.yml

$ git push origin TECH-2871-improve-cicd-charts
Everything up-to-date
```